relay/telemetry.go

```diff
@@ -0,0 +1,57 @@
+package relay
+
+import (
+	"fmt"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/metric"
+
+	"context"
+	"sync"
+)
+
+type Int64Gauge struct {
+	gauge   metric.Int64Gauge
+	options []metric.RecordOption
+	mutex   sync.Mutex
+}
+
+var (
+	tracer = otel.Tracer("github.com/datachainlab/lcp-go/relay")
+	meter  = otel.Meter("github.com/datachainlab/lcp-go/relay")
+)
+
+const (
+	namespaceRoot = "lcp-go"
```

Comment:
I think it is uncommon to use hyphens in a metric name.

> For each multi-word dot-delimited component of the name separate the words by underscores (i.e. use snake_case). For example http.response.status_code denotes the status code in the http namespace.

https://opentelemetry.io/docs/specs/semconv/general/naming/

Additionally, it might be better to use the word relayer for the root namespace, which is used [here](https://github.com/hyperledger-labs/yui-relayer/blob/e5f6d77adf60a6bd8ad1a01d5c667a39492ac80d/internal/telemetry/metrics.go#L15):

```suggestion
	namespaceRoot = "relayer.lcp_go"
```

Reply:
renamed